pkg/pub_dartdoc_data/lib/dartdoc_page.dart

```diff
@@ -52,13 +52,22 @@ final class DartDocPage {
   /// Sanitized HTML for the [content] pane.
   final String content;
 
+  final String? baseHref;
+  final String? usingBaseHref;
+  final String? aboveSidebarUrl;
+  final String? belowSidebarUrl;
```

Comment:
If you look in the documentation string for `DartDocPage` there is no sidebars above or below.

Also can we have documentation comments for these.

If they are used by client side to load HTML files, then clearly we should explain this.